src/model/bot.ts

```diff
@@ -0,0 +1,4 @@
+export enum BotStatusEnum {
+  ACTIVE = 1,
+  ARCHIVED = 2,
```

Comment:
Let's use CamelCase for consistentcy

```suggestion
  Archived = 2,
```